spinalcordtoolbox/reports/qc2.py

```diff
@@ -765,7 +774,7 @@ def mosaic(img: Image, centers: np.ndarray, radius: tuple[int, int] = (15, 15),
     for center, slice in zip(centers.astype(int), img.data):
         # Add a margin before cropping, in case the center is too close to the edge
         # Also, use Kronecker product to scale each block in multiples
-        cropped.append(np.kron(np.pad(slice, radius)[
+        cropped.append(np.kron(np.pad(slice, ((radius[0], radius[0]), (radius[1], radius[1])))[
```

Comment:
If you want to avoid the repetition, this should also work:
```suggestion
        cropped.append(np.kron(np.pad(slice, [[r] for r in radius])[
```
Proof of concept:
```py
>>> import numpy as np
>>> data = np.arange(12, 36).reshape((4, 6)); data
array([[12, 13, 14, 15, 16, 17],
       [18, 19, 20, 21, 22, 23],
       [24, 25, 26, 27, 28, 29],
       [30, 31, 32, 33, 34, 35]])
>>> radius = (1, 2)
>>> np.pad(data, radius)  # bad
array([[ 0,  0,  0,  0,  0,  0,  0,  0,  0],
       [ 0, 12, 13, 14, 15, 16, 17,  0,  0],
       [ 0, 18, 19, 20, 21, 22, 23,  0,  0],
       [ 0, 24, 25, 26, 27, 28, 29,  0,  0],
       [ 0, 30, 31, 32, 33, 34, 35,  0,  0],
       [ 0,  0,  0,  0,  0,  0,  0,  0,  0],
       [ 0,  0,  0,  0,  0,  0,  0,  0,  0]])
>>> np.pad(data, [[r] for r in radius])  # good
array([[ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0],
       [ 0,  0, 12, 13, 14, 15, 16, 17,  0,  0],
       [ 0,  0, 18, 19, 20, 21, 22, 23,  0,  0],
       [ 0,  0, 24, 25, 26, 27, 28, 29,  0,  0],
       [ 0,  0, 30, 31, 32, 33, 34, 35,  0,  0],
       [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0]])
```